public/app/features/canvas/runtime/element.tsx

```diff
@@ -486,6 +486,24 @@ export class ElementState implements LayerElement {
   };
 
   onElementClick = (event: React.MouseEvent) => {
+    this.onTooltipCallback();
+  };
+
+  onElementKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'Enter') {
+      const scene = this.getScene();
+      if (scene?.tooltipCallback && scene.tooltip?.anchorPoint) {
+        const rect = this.div?.getBoundingClientRect();
+        scene.tooltipCallback({
+          anchorPoint: { x: rect?.right ?? scene.tooltip.anchorPoint.x, y: rect?.top ?? scene.tooltip.anchorPoint.y },
+          element: this,
+          isOpen: true,
+        });
+      }
+    }
+  };
```

Comment:
This is interesting - makes each canvas element tabbable, though as a user when pressing "enter" I might expect that action would "select" the currently tabbed element 🤔 (which is what click does) Maybe we should expand the onElementKeyDown function to also select the current tabbed selection?